homework_no_git/js/app.js

```diff
@@ -0,0 +1,116 @@
+
+// Section 1
+// 1. Parameter - Variable defined in the function's declaration when the function is defined.
+// // Argument - Actual values passed into the function when the function is called.
+
+// 2. return - Allows us to "hand back" a value to the line where we called the function.
+// // console.log - Simply calls the function, and lets us know whether it's working or not.
```

Comment:
all console.log() does is print whatever you pass in to it on the console.  that's it.  it may or may not call something but it's simply a tool for developers to use for debugging.